development/webpack/webpack.web-embed.config.js

```
@@ -1,11 +1,181 @@
 const { merge } = require('webpack-merge');
 
+const HtmlWebpackPlugin = require('html-webpack-plugin');
 const path = require('path');
```

Biome (1.9.4)

[error] A Node.js builtin module should be imported with the node: protocol.

Using the node: protocol is more explicit and signals that the imported module belongs to Node.js.
Unsafe fix: Add the node: protocol.


(lint/style/useNodejsImportProtocol)